### DISCUSSION TITLE
13-group Question

### DISCUSSION THREAD

**Discussion by Martin7-1** [courses-at-nju-by-hfwei/discrete-math-problem-sets#11] [Q&A] [answered]
老师好！
老师可以在仔细说明一下如何证明 **u 是 a 的逆元** 吗？（如下图所示，感觉听课程录屏还是没听懂）
![image](https://user-images.githubusercontent.com/77151639/123357275-cfed1380-d59b-11eb-8864-5768eda1023c.png)

**Comment by Martin7-1** [comment, marked as answer]
补充：蓝色字体是为了证明 **ua = 1 mod m**，这个我理解了，那么知道这个为什么可以证得 **逆元** 呢

**Reply by hengxin** [reply]
@Martin7-1 根据逆元的定义, a 与 u 互为逆元的定义就是 ua = au = e, e 是单位元, 在这里单位元就是 1。这里就是在验证 ua = au = 1。

**Reply by Martin7-1** [reply]
明白了....所以 **模 m 剩余类乘法群** 指的是 (G, \*)，其中 G 是从 1 到 m-1 的元素中与 m 互素的元素构成的集合，\* 指的是乘法后模m吗

**Reply by hengxin** [reply]
@Martin7-1 是的

**Reply by Martin7-1** [reply]
谢谢老师，老师保佑我及格
